src/DynamicListForm.jsx

```diff
@@ -65,6 +70,16 @@
                                     additem: this.addItem,
                                     options: this.props.options,
                                     itemCount: Object.keys(dialogValues.list).length,
+                                    validationFailed: validationFailed && validationFailed[idx],
+                                    onValidationChange: value => {
+                                        const delta = validationFailed ? [...validationFailed] : [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230725-122305-68cb803a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)